ui/raidboss/data/05-shb/trial/diamond_weapon-ex.js

```diff
@@ -0,0 +1,268 @@
+import ZoneId from '../../../../../resources/zone_id';
+import NetRegexes from '../../../../../resources/netregexes';
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
+
+export default {
+  zoneId: ZoneId.TheCloudDeckExtreme,
+  timelineFile: 'diamond_weapon-ex.txt',
+  triggers: [
+    // Phase 1&3
+    {
+      id: 'DiamondEx Diamond Rain',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5FA7', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Diamant-Waffe', id: '5FA7', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Diamant', id: '5FA7', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'ダイヤウェポン', id: '5FA7', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+      run: (data) => data.phase = data.phase || 1,
+    },
+    // @TODO: There's probably a better callout for these mechanics, e.g.
+    // `jump -> stay -> aoe`
+    // `stay -> jump -> aoe`
+    // `jump -> stack -> stay`
+    // `stay -> stack -> jump`
+    // `jump -> spread -> stay`
+    // `stay -> spread -> jump`
+    // but need to figure out what side player is on
+    {
+      id: 'DiamondEx Adamant Purge West Diamond Rain',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5F9B', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Diamant-Waffe', id: '5F9B', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Diamant', id: '5F9B', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'ダイヤウェポン', id: '5F9B', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'West claw -> AoE',
```

Comment:
Yeah, I think switch/stay would be nice to do, but in the meantime could you phrase these positively vs where the thing to avoid is, e.g. `go east -> aoe`.